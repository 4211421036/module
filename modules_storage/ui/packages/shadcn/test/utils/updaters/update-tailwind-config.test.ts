version https://git-lfs.github.com/spec/v1
oid sha256:93f6e5b8fc74e7e0034703f6362b208a9dcf0daff3496c3884170a6abd899e69
size 34092
