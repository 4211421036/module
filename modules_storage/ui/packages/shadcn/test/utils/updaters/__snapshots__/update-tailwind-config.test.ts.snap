version https://git-lfs.github.com/spec/v1
oid sha256:39506231952217b525700b280d2a12fd2fdea01acd2efbf925afe7edd8f1f3ad
size 13818
